src/core/internal/index.ts

```diff
@@ -157,6 +157,11 @@ export class FrameElement {
       this.focusChange(true);
     });
     this.iFrameFormElement.on(ELEMENT_EVENTS_TO_CLIENT.BLUR, (state) => {
+      if (state.value && this.iFrameFormElement.fieldType === ELEMENTS.EXPIRATION_MONTH.name) {
+        if (state.value.length === 1 && Number(state.value) < 10) {
+          this.iFrameFormElement.setValue(`0${state.value}`);
```

Comment:
zero should be appended on onchange not on onblur . Right ?

Reply:
Done. Now for number > 2 added it on change and for 1 it is onblur